src/server/auth/cmds/cmds.go

```diff
@@ -16,43 +16,56 @@ import (
 
 var githubAuthLink = `https://github.com/login/oauth/authorize?client_id=d3481e92b4f09ea74ff8&redirect_uri=https%3A%2F%2Fpachyderm.io%2Flogin-hook%2Fdisplay-token.html`
 
-// ActivateCmd returns a cobra.Command to activate the security features of
-// Pachyderm within a Pachyderm cluster. All repos will go from
-// publicly-accessible to accessible only by the owner, who can subsequently add
-// users
+// ActivateCmd returns a cobra.Command to enable Pachyderm's auth system
 func ActivateCmd() *cobra.Command {
 	var admins []string
-	activate := &cobra.Command{
-		Use: "activate activation-code",
-		Short: "Activate the security features of pachyderm with an activation " +
-			"code",
-		Long: "Activate the security features of pachyderm with an activation " +
-			"code",
+	enableAuth := &cobra.Command{
+		Use:   "enable --admins=[admins...]",
```

Comment:
It's maybe a bit weird that above we use `activate` for the process of activating auth but here we use `enable`. Would unifying on one word make sense to you? I'm assuming you had some reasons for switching from activate to enable here.